Sources/SwiftGen/Commander/ConfigCommands.swift

```diff
@@ -153,7 +131,58 @@ enum ConfigCLI {
   }
 }
 
-// MARK: Private
+// MARK: - Parallel commands
+
+private extension Config {
+  func runCommands(verbose: Bool) throws {
+    let commandsAndEntries = try collectCommandsAndEntries()
+
+    let errors = commandsAndEntries.parallelCompactMap { cmd, entry -> Swift.Error? in
+      do {
+        try run(cmd: cmd, entry: entry, verbose: verbose)
+        return nil
+      } catch {
+        return error
+      }
+    }
+
+    if errors.count == 1 {
+      throw errors[0]
+    } else if errors.count > 1 {
+      throw Error.multipleErrors(errors)
+    }
+  }
+
+  func run(cmd: ParserCLI, entry: ConfigEntry, verbose: Bool) throws {
+    var entry = entry
+
+    entry.makingRelativeTo(inputDir: inputDir, outputDir: outputDir)
+    if verbose {
+      for item in entry.commandLine(forCommand: cmd.name) {
+        logMessage(.info, " $ \(item)")
+      }
+    }
+
+    try entry.checkPaths()
+    try entry.run(parserCommand: cmd)
+  }
+
+  /// Flatten all commands and their corresponding entries into 1 list
+  func collectCommandsAndEntries() throws -> [(ParserCLI, ConfigEntry)] {
+    try commands.keys.sorted()
+      .compactMap { cmd in
+        guard let parserCmd = ParserCLI.command(named: cmd) else {
+          throw Config.Error.missingEntry(key: cmd)
```

Comment:
I know that you didn't change that part and that's the error we already used before that PR but I'm wondering if that's the right one to throw? If a user is referring to a parser name that doesn't actually exist, it's not really that we're missing an entry for that key but rather that the key is invalid, right?